GitCommands/Git/GitModule.cs

```diff
@@ -3850,7 +3850,7 @@ public bool HasLfsSupport()
 
         public bool StopTrackingFile(string filename)
         {
-            return _gitExecutable.Execute($"rm --cached {filename}").ExitedSuccessfully;
+            return _gitExecutable.Execute($"rm --cached {filename.ToPosixPath().Quote()}").ExitedSuccessfully;
         }
```

Comment:
Need to make this use GitArgumentBuilder. 

@drewnoakes. Might need to run through and find all code like this.

Reply:
Yes. I just saw it here and just wanted to bring it up.

Reply:
The PR that introduced `GitArgumentBuilder` mentioned that we aren't using it everywhere we _could_. It's not strictly necessary until we start needing the configuration overrides it provides. Of course there's nothing wrong with introducing it here and elsewhere.